impl/src/templates/remote_crates.bzl.template

```diff
@@ -16,8 +16,9 @@ def _new_git_repository(name, **kwargs):
         new_git_repository(name=name, **kwargs)
 
 def {{workspace.gen_workspace_prefix}}_fetch_remote_crates():
-{% for crate in crates %}
-    {%- if crate.source_details.git_data %}
+{%- if crates|length %}
```

Comment:
I think this adds a third way of checking the length.  I think we [already have this](https://github.com/google/cargo-raze/blob/master/impl/src/templates/partials/common_attrs.template#L4):

```python
{%- if crate.aliased_dependencies | length != 0 %}
```
[and this](https://github.com/google/cargo-raze/blob/master/impl/src/templates/partials/common_attrs.template#L34)
```python
{%- if crate.aliased_dependencies %}
```

Would you mind changing this to
```suggestion
{%- if crates %}
```
in order to consolidate?  (assuming I haven't missed something)